docs/_tutorials/reference.md

```diff
@@ -0,0 +1,141 @@
+| `body` | All listeners | Object that contains the entire body of the request (superset of `payload`). Some accessory data is only available outside of the payload (such as `trigger_id` and `authorizations`).
+
+#### Body and payload references
+The structure of the `payload` and `body` is detailed on the API site:
+- `action`: [`body`](https://api.slack.com/reference/interaction-payloads/block-actions) and [`payload`s](https://api.slack.com/reference/block-kit/block-elements)
+- `event`: [`body`](https://api.slack.com/types/event) and [`payload`s](https://api.slack.com/events)
+- `shortcut`: [`body` and `payload`](https://api.slack.com/reference/interaction-payloads/shortcuts)
+- `command`: [`body` ](https://api.slack.com/interactivity/slash-commands)
+- `view`: [`view_submission` `body` and `payload`](https://api.slack.com/reference/interaction-payloads/views#view_submission); [`view_closed` `body` and `payload`](https://api.slack.com/reference/interaction-payloads/views#view_closed)
+- `options`: [`body` and `payload`](https://api.slack.com/reference/block-kit/block-elements#external_select)
+
+### Difference from listener middleware
+Listener middleware is used to implement logic across many listener functions (though usually not all of them). Listener middleware has the same arguments as the above listener functions, with one distinction: they also have a `next()` function that **must** be called in order to pass the chain of execution. Learn more about listener middleware [in the documentation](/bolt-js/concepts#listener-middleware).
+
+## Initialization options
+Bolt includes a collection of initialization options to customize apps. There are two primary kinds of options: Bolt app options and receiver options. The receiver options may change based on the receiver your app uses. The following receiver options are for the default `ExpressReceiver` (so they'll work as long as you aren't using a custom receiver).
+
+### Receiver options
+`ExpressReceiver` options can be passed into the app constructor, just like the Bolt app options. They'll be passed to the `ExpressReceiver` instance upon initialization.
+
+| Option  | Description  |
+| :---: | :--- |
+| `signingSecret` | A `string` from your app's configuration (under "Basic Information") which verifies that incoming events are coming from Slack |
+| `endpoints` | A `string` or `object` that specifies the endpoint(s) that the receiver will listen for incoming requests from Slack. For `object`, the `key` is the type of events (ex: `events`), and the value is the endpoint (ex: `/myapp/events`). **By default, all events are sent to the `/slack/events` endpoint** |
+| `processBeforeResponse` | `boolean` that determines whether Events API events should be automatically acknowledged. This is primarily useful when running apps on FaaS to ensure events listeners don't unexpectedly terminate. Defaults to `true`.  |
```

Comment:
I'd love to be able to update the [Deploying to AWS Lambda - Prepare the app](https://github.com/slackapi/bolt-js/blob/main/docs/_deployments/aws-lambda.md#prepare-the-app) section to link to the `processBeforeResponse` documentation. 🙏🏻